pkg/store/kotsstore/version_store.go

```diff
@@ -1007,3 +1014,54 @@ func (s *KOTSStore) GetCurrentUpdateCursor(appID string, channelID string) (stri
 
 	return updateCursor.String, versionLabel.String, nil
 }
+
+func (s *KOTSStore) HasStrictPreflights(appID string, sequence int64) (bool, error) {
+
```

Comment:
let's stick to the pattern of not having empty lines at the beginning of functions? :)